requirements/centos7.requirements.txt

```diff
@@ -0,0 +1,4 @@
+pylint==1.9.5
+astroid==1.6.6
+pytest==4.6.11
```

Comment:
```
ERROR: pip's legacy dependency resolver does not consider dependency conflicts when selecting packages. This behaviour is the source of the following dependency conflicts.
pytest 4.6.11 requires six>=1.10.0, but you'll have six 1.9.0 which is incompatible.
```
It's because the latest python-six package available in centos 7 (installed per the Dockerfiles/centos7.Dockerfile) is python-six-1.9.0-2.el7.
I suppose you need to add six>=1.10.0 to this requirements file.

Reply:
This is interesting. I can successfully build the cento7 container (which installs dependencies from the cento7.requirements.txt). I will create a short meeting on Monday to look into this (it seems) docker issues.

Reply:
Still the same issue. Or even worse now:
```
ERROR: pip's legacy dependency resolver does not consider dependency conflicts when selecting packages. This behaviour is the source of the following dependency conflicts.
sphinx-autobuild 0.7.1 requires PyYAML>=3.10, which is not installed.
pytest 4.6.11 requires six>=1.10.0, but you'll have six 1.9.0 which is incompatible.
```
IMHO this isn't much about containers but about virtualenv behaviour.

Reply:
It seems the tests work fine even with this error :) So let it be there.
BTW, I've tried older versions of pytest, down to 3.9.1, and even this one requires six>=1.10.0.